### ISSUE TITLE
finetune with own data

### ISSUE THREAD

**Issue by Nidhogg-Mzy** [bowang-lab/ECG-FM#6]
Hi authors, really appericiate your work. I want to use your checkpoint to fine-tune my own ECG data, but it lacks the information that your data preprocessing steps need(I just have ECG signal and labels), so could you please give some hints on how to use the self-defined dataset to fine-tune the model? I am a little bit confused by the dataset definition code in your training script. Thank you!

**Comment by ndhuynh02** [comment]
I am also wondering how this model is finetuned. As stated in the paper, the model only takes in 5s of the ECG signal. Does this mean I need to crop my data, if so, what segment should be chosen. 
Additionally, I have another question in the finetuned checkpoint is provided. Does the Physionet2021 dataset get masked like in the pre-training phase? Or the full 12-lead ECG signal is used?

**Comment by KadenMc** [comment]
Hi all, have you referred to this README: https://github.com/Jwoo5/fairseq-signals/tree/master/scripts/preprocess/ecg

If not, this should answer your questions! If it does, please close the issue. If not, feel free to ask further questions here.
